### PR TITLE
Littletweaks

### PR DESCRIPTION
**Pull request by astroyouth**
some tweaks from the main PR coments.

### CODE REVIEW

**Comment by ccfredbaum** [comment]
not sure about the shortcode change as we are still rendering CC as a backup for JustCoach

File: my-video-room/core/class-sitedefaults.php [hunk 1]
```diff
@@ -74,7 +73,7 @@ public function core_menu_setup() {
 	 * Render Module Menu.
 	 */
 	public function core_menu_button() {
-		$name = self::MODULE_CORE_DISPLAY;
+		$name = self::esc_html_e( 'Video Default Settings', 'my-video-room' );
```
Comment (alecsammon) [inline my-video-room/core/class-sitedefaults.php:76]: this is wrong!

`$name = esc_html__( 'Video Default Settings', 'my-video-room' );`

File: my-video-room/module/security/templates/class-securitybuttons.php [hunk 2]
```diff
@@ -50,28 +49,28 @@ public static function site_wide_enabled( $input_type = null ) {
 			// Rendering Output.
 		if ( $site_override ) {
 			if ( 'nourl' === $input_type ) {
-				$output .= '<a class="button button-primary" style="background-color:#daab33">Site Enforcement Active</a>';
+				$output .= '<a class="button button-primary" style="background-color:#daab33">' . esc_html_e( 'Site Enforcement Active', 'my-video-room' ) . '</a>';
```
Comment (alecsammon) [inline my-video-room/module/security/templates/class-securitybuttons.php:52]: as these ones are in a php string then they should not be echo'ed so you need to use `esc_html__` and not `esc_html_e`.

This is throughout this file.

**Review by alecsammon — CHANGES_REQUESTED** [review]
Please see inline comments

File: my-video-room/module/security/templates/class-securitybuttons.php (hunk 2 above)
Comment (astroyouth) [reply my-video-room/module/security/templates/class-securitybuttons.php:52]: corrected. 
waiting for a couple of other changes for push and PR

**Comment by astroyouth** [comment]
changes done.

File: my-video-room/core/class-sitedefaults.php [hunk 3]
```diff
@@ -74,7 +73,7 @@ public function core_menu_setup() {
 	 * Render Module Menu.
 	 */
 	public function core_menu_button() {
-		$name = self::MODULE_CORE_DISPLAY;
+		$name = self::esc_html__( 'Video Default Settings', 'my-video-room' );
```
Comment (alecsammon) [inline my-video-room/core/class-sitedefaults.php:76]: this is still wrong. you need to remove the `self::`

**Review by alecsammon — APPROVED** [review]

**Comment by alecsammon** [comment]
@astroyouth  - something has gone weird with this. Did you merge Fred's branch into it?

**Comment by astroyouth** [comment]
Only did pull requests I wouldn't dare do a merge.

**Comment by alecsammon** [comment]
Ok - I will take a look

**Comment by astroyouth** [comment]
Willco.

**Comment by alecsammon** [comment]
Most of these have already been merged - I've taken the remaining ones onto a new branch.

**Pull request closed without merging**